### ISSUE TITLE
4.4.* version can not be uploaded to App Store Connect

### ISSUE THREAD

**Issue by JustasL** [CartoDB/mobile-sdk#454]
Hi, we can not upload iOS build to App Store Connect when we are using 4.4.* SDK version

Issue seems to be with Bitcode (our project uses Bitcode)

Issue can be easily reproduced by uploading build with Carto SDK (installed via Carthage or SPM)

Here's what Apple replies to us when trying to upload build:

ITMS-90562: Invalid Bundle - The app cannot be processed because options not allowed to be embedded in bitcode are detected in the submission. It is likely that you are not building the app with the toolchain provided in Xcode. Rebuild your entire app with the latest GM Xcode and submit the app again.

**Comment by mtehver** [comment]
Thanks for reporting this. Are you using 'Metal' variant of the SDK or 'normal' variant?

We did not really change bitcode related compilation flags in SDK 4.4.x vs 4.3.x but we package SDK now as xcframework, not as fat framework (as we did previously). We will investigate this.

**Comment by JustasL** [comment]
I think we are using `normal` variant, not sure how to check it?

With CocoaPods I can see I there are different pods:
```
pod 'CartoMobileSDK', '4.4.1'
pod 'CartoMobileSDK-Metal', '4.4.1'

```  
  
But with Carthage or SPM I can find an option to chose metal version

**Comment by mtehver** [comment]
@JustasL Thanks for clarifying, this means you are using 'normal' variant based on Apple OpenGL ES renderer.

Regarding the issue, we are currently actively looking into it, will post an update once we have found a solution.

**Comment by mtehver** [comment]
Thanks again for reporting this, I have pushed a hotfix version (4.4.1-hotfix1) to https://github.com/nutiteq/mobile-sdk-ios-swift-package, you need to use 'exact version' and '4.4.1-hotfix1' when adding or replacing the existing package.

Turned out to be a floating point optimization flag that was not supported during bitcode recompilation phase.

I will replace Cocoapods with fixed versions tomorrow.

**Comment by JustasL** [comment]
Can confirm - app was successfully uploaded to App Store using `4.4.1-hotfix1`

**Comment by mtehver** [comment]
New cocoapods are available now, version 4.4.1.1 includes the hotfix.
